pkg/deploy/controller/deployment/controller.go

```diff
@@ -20,6 +20,21 @@ import (
 	"github.com/openshift/origin/pkg/util"
 )
 
+// maxRetryCount is the maximum number of times the controller will retry errors.
+// The first requeue is after 5ms and subsequent requeues grow exponentially.
+// This effectively can extend up to 5^10ms which caps to 1000s:
+//
+// 5ms, 25ms, 125ms, 625ms, 3s, 16s, 78s, 390s, 1000s, 1000s
```

Comment:
Not configurable currently. See [here](https://github.com/kubernetes/kubernetes/blob/7d040dd12854795ed63115fb0d95facdc39bd60c/staging/src/k8s.io/client-go/util/workqueue/default_rate_limiters.go#L41).